db/seeds.rb

```diff
@@ -6,11 +6,13 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-db = Specialty.create(name: "Database", url_name: "database")
-fe = Specialty.create(name: "Front End", url_name: "front_end")
-be = Specialty.create(name: "Back End", url_name: "back_end")
+db = Specialty.create(name: "Database", url_name: "database", image: File.new("#{Rails.root}/app/assets/images/database/horse_animal_totem.png"))
+fe = Specialty.create(name: "Front End", url_name: "front_end", image: File.new("#{Rails.root}/app/assets/images/back_end/Wolf_animal_totem.png"))
+be = Specialty.create(name: "Back End", url_name: "back_end", image: File.new("#{Rails.root}/app/assets/images/front_end/hawk_animal_totem.png"))
 
-Developer.create(name: "JP", last_name: "Perry", email: "hotdog1@hotmail.com", password: "$", rate: 10, specialty_id: db.id, profile_picture: "https://media.licdn.com/mpr/mpr/shrinknp_200_200/AAEAAQAAAAAAAAZvAAAAJDFiY2JjNTc4LTQ0NDktNDcyYy05YzMxLTcyZmVmZDU3OTU5OA.jpg")
-Developer.create(name: "Steve", last_name: "Pentler", email: "hotdog2@hotmail.com", password: "$", rate: 20, specialty_id: fe.id, profile_picture: "https://media.licdn.com/mpr/mpr/shrinknp_200_200/AAEAAQAAAAAAAAZvAAAAJDFiY2JjNTc4LTQ0NDktNDcyYy05YzMxLTcyZmVmZDU3OTU5OA.jpg")
-Developer.create(name: "Hector", last_name: "Huertas", email: "hotdog3@hotmail.com", password: "$", rate: 30, specialty_id: be.id, profile_picture: "https://media.licdn.com/mpr/mpr/shrinknp_200_200/AAEAAQAAAAAAAAZvAAAAJDFiY2JjNTc4LTQ0NDktNDcyYy05YzMxLTcyZmVmZDU3OTU5OA.jpg")
-Developer.create(name: "Aaron", last_name: "Greenspan", email: "hotdog4@hotmail.com", password: "$", rate: 40, specialty_id: be.id, profile_picture: "https://media.licdn.com/mpr/mpr/shrinknp_200_200/AAEAAQAAAAAAAAZvAAAAJDFiY2JjNTc4LTQ0NDktNDcyYy05YzMxLTcyZmVmZDU3OTU5OA.jpg")
+Developer.create(name: "JP", last_name: "Perry", email: "hotdog1@hotmail.com", password: "$", rate: 10, specialty_id: db.id, skills_rating: 2 )
+Developer.create(name: "Steve", last_name: "Pentler", email: "hotdog2@hotmail.com", password: "$", rate: 20, specialty_id: fe.id, skills_rating: 3 )
+Developer.create(name: "Hector", last_name: "Huertas", email: "hotdog3@hotmail.com", password: "$", rate: 30, specialty_id: be.id, skills_rating: 4 )
+Developer.create(name: "Aaron", last_name: "Greenspan", email: "hotdog4@hotmail.com", password: "$", rate: 40, specialty_id: be.id )
+
+Admin.create(username: 'admin', password: 'admin', email: 'admin')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.